### PR TITLE
README: document archiving and direct issues to gonum/gonum

### PR DESCRIPTION
**Pull request by kortschak**
Please take a look.

If this is OK, I'll archive this repo and send similar changes to the other deprecated repos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,10 +6,10 @@ This is a matrix package for the Go language.
 
 ## Issues
 
-If you find any bugs, feel free to file an issue on the github issue tracker. Discussions on API changes, added features, code review, or similar requests are preferred on the gonum-dev Google Group.
+If you find any bugs, feel free to file an issue on the github issue tracker for gonum/gonum if the bug exists in that reposity; no code changes will be made to this repository. Other dicussions should be taken to the gonum-dev Google Group.
```
Comment (sbinet) [inline README.md]: we may want to directly point people at the `gonum/gonum` issue tracker:
`[gonum/gonum](https://github.com/gonum/gonum/issues)`

Comment (kortschak) [reply README.md]: Done

**Comment by kortschak** [comment]
@btracey Can I have your approval too?

**Comment by kortschak** [comment]
Thanks. I think that phrasing is OK given the difficult philosophical nature of continuities.

**Pull request merged**